hardcore_reset.py

```diff
@@ -27,6 +30,34 @@ def hard_reset(skip = False):
     if skip == False:
         print('Progress deleted!\n')
         print('Creating space for a new database....')
+    c.execute("CREATE TABLE `inventory` ('id' INTEGER NOT NULL, \

```

Comment:
should this `\^M` be here

Reply:
I Googled it. It... says it's got to be with how different software uses different end of line characters... Idunno.